global.variables.yml

```diff
@@ -1,4 +1,37 @@
 variables:
+  ###########################################################################################################################
```

Comment:
Can we still use ADO variable groups with this change?

Reply:
Yes. It's not impacting existing ADO capabilities

Reply:
Shouldn't ADO variables be declared with 
```
- name: ..
  value: ..
``` 
to support variable groups? Just trying to fully understand the implications of this. If that is still the case, we should probably mention that in the docs. For now we only have one variable group declared in all ADO pipelines, i.e. 
```
- group: 'PLATFORM_VARIABLES'
``` 
the one containing pipeline secrets. Similarly, in case other groups are needed going forward, they cannot be added to the global.variables.yml file. They need to be added to pipelines instead. Would you agree?

Reply:
Not sure I understand the question. From ADO perspective, we continue to use the same methods, whether its pipeline variables, variable groups or variables in templates. Only change is the path of the template YAML to live outside of the ADO folder

Reply:
As discussed: As long as we don't mix the short syntax 

```yaml
<key>: <value>
```

with the 'long syntax'

```yaml
- name: <key>
  value: <value>
```
we should be fine

Reply:
Yes, addressed offline today, we can solve it.